doc/user_manual.md

```diff
@@ -0,0 +1,1009 @@
+    # capping groups) is attached on APs of a specific class.
+    # In this example, no fragment will be attached to APs of class apclass2:1
+    RCN apclass2:1 none
+    …
+    # Capping groups
+    CAP apclass1:0 cap1:0
+    CAP apclass1:0 cap2:0
+    …
+    # Forbidden ends
+    DEL apclass1:0
+
+## Ring Closures Compatibility Matrix file
+
+The ring closures (RC) compatibility matrix file specifies the pairs of AP classes that are allowed to form ring-closing bonds. Note that, contrary to the standard compatibility matrix, the RC compatibility matrix is symmetric. The syntax is the same as for a general-purpose compatibility matrix.
+
+    # Comment line
+    RCN apclass1:0 apclass1:1,apclass2:0,
+    RCN apclass1:1 apclass1:0
+    …
+
+## Initial Population File
+
+SDF format is used to provide an initial set of fully characterized candidates (i.e., initial population). To be properly interpreted by DENOPTIM this SDF file must include the following tags:
+
+*   `<FITNESS>` tag specifying the numerical fitness value.
+*   `<UID>` tag specifying the unique identifier of the chemical entity (often the InChiKey).
+*   `<GraphJson>` tag specifying the essential part of the graph representation of the chemical entity.
+
+## Candidate Chemical Entity Files  {#CandidateChemicalEntityFiles}
+
+SDF format file used to communicate the definition of a candidate entity to external fitness providers (see [Fitness evaluation](#FitnessEvaluation)).
+
+DENOPTIM programs output a non-optimized 3D-structure of the chemical entity in SDF format together with its graph representation (tag `<GraphJson>`) and additional information (i.e., SMILES, InChiKey, unique identifier). By default, the Cartesian coordinates reported in these files are those of a three-dimensional tree-like model built by aligning each building block according to the attachment point vectors. The resulting geometry is not refined, and is only meant to facilitate further processing and visual inspection. While such geometry can be used as starting point for further molecular modeling, it should never be used to calculate any property dependent on the molecular geometry. The alignment of the building blocks is skipped when using the `FP-No3dTreeModel` keyword. In this case, the SDF file will contain Cartesian coordinates of the initial building blocks, and will therefore be characterized by a correct topology but a non-sense arrangement of the atoms.
+
+## Fitness Files {#FitnessFiles}
+
+SDF format files used to store the final results for an evaluated chemical entity and to receive the fitness value from any external fitness provider. These files must contain one of these two tags:
+
+*   `<MOL_ERROR>` tag specifying that the candidate entity cannot be coupled with a fitness value. The content of this field is used to report details on the reasons leading to this result. The following syntax must be used:   
+
+        …
+        > <Mol\_ERROR>
```

Comment:
fix wrong use of backslashes

Reply:
@marcellocostamagna starting from here I stop pinging the same errors with backslashes because there are too many, but the must be fixed throughout this and all other markdown files!